web/documentserver-example/python/src/views/actions.py

```diff
@@ -459,6 +455,26 @@ def downloadhistory(request):
         response.setdefault('error', 'File not found')
         return HttpResponse(json.dumps(response), content_type='application/json', status=404)
 
+def historyobj(request):
+    body = json.loads(request.body)
+    response = {}
+    fileName = None
```

Comment:
In Python, they walk like snakes. 🐍
See [PEP 8](https://peps.python.org/pep-0008/#function-and-variable-names).

Reply:
> In Python, they walk like snakes. 🐍 See [PEP 8](https://peps.python.org/pep-0008/#function-and-variable-names).

Python example contains all naming styles so it doesn't matter without refactoring.

Reply:
What is the point of writing a new one and intentionally making mistakes in it just to create more work for the person who will have to edit it in the future? You're right, our example contains different code styles. However, this doesn't mean that we should write new code carelessly.

Reply:
@sshakndr, you mark it as resolved, but it isn't.